presto-parser/src/main/java/io/prestosql/sql/parser/ErrorHandler.java

```diff
@@ -182,6 +182,12 @@ public Analyzer(
                 int tokenIndex = current.tokenIndex;
                 CallerContext caller = current.caller;
 
+                while (tokenIndex < stream.size() && stream.get(tokenIndex).getType() == SqlBaseParser.WS) {
```

Comment:
- `tokenIndex < stream.size()` is redundant
- `SqlBaseParser.WS` should be provided as configuration